react/components/EXPERIMENTAL_Table/Sections/Cell.tsx

```diff
@@ -1,53 +1,82 @@
 import React, {
-  FC,
   createContext,
   useContext,
   useState,
   PropsWithChildren,
   CSSProperties,
   DetailedHTMLProps,
   HTMLAttributes,
+  Ref,
+  forwardRef,
+  FC,
 } from 'react'
 import classNames from 'classnames'
 
 import CaretDown from '../../icon/CaretDown/index.js'
 import CaretUp from '../../icon/CaretUp/index.js'
-
-const SUFIX_GAP = 0.5
+import { ComposableWithRef } from '../types'
 
 const HoverContext = createContext<boolean>(false)
 
-function HoverProvider({ children, value }) {
-  return <HoverContext.Provider value={value}>{children}</HoverContext.Provider>
+export enum CellTag {
```

Comment:
There is a discussion about using `const enum` instead of `enum`, not sure if it applies in this case but I would check it out

Reply:
I'm keeping the `enum` because they will only be relevant for `graphql` in the future. (also, we need additional plugins on babel for that 😅)